delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/FileExtraFields.scala

```diff
@@ -0,0 +1,61 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.files.model
+
+import akka.http.scaladsl.model.ContentType
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.instances._
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.files.model.FileAttributes.FileAttributesOrigin
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.files.model.FileEvent._
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageType
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import io.circe.Encoder
+import io.circe.generic.semiauto.deriveEncoder
+
+case class FileExtraFields(
+    storage: Iri,
+    storageType: StorageType,
+    physicalFile: Option[Int],
+    bytes: Option[Long],
+    mediaType: Option[ContentType],
+    origin: Option[FileAttributesOrigin]
+)
+
+object FileExtraFields {
+  def fromEvent(event: FileEvent): FileExtraFields =
```

Comment:
This must match this implementation:
https://github.com/BlueBrain/nexus/blob/master/delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/FileEventExchange.scala#L51